<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReadingTrainer</class>
 <widget class="QWidget" name="ReadingTrainer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Reading practice</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Type the correct reading for this word:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="writingLabel">
     <property name="font">
      <font>
       <pointsize>20</pointsize>
      </font>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLineEdit" name="userInput"/>
     </item>
     <item>
      <widget class="QPushButton" name="nextButton">
       <property name="text">
        <string>&amp;Next</string>
       </property>
       <property name="autoDefault">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="okButton">
       <property name="text">
        <string>&amp;Ok!</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="resultLabel">
     <property name="text">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="ToolBarDetailedView" name="detailedView"/>
   </item>
   <item>
    <widget class="QLabel" name="statusLabel">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ToolBarDetailedView</class>
   <extends>QTextBrowser</extends>
   <header>ToolBarDetailedView.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>userInput</sender>
   <signal>returnPressed()</signal>
   <receiver>okButton</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>264</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>264</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
